pkg/web_css/lib/src/_base.scss

```diff
@@ -46,12 +46,12 @@ body,
     font-weight: 400;
   }
 
-  h1 { font-size: 36px; }
-  h2 { font-size: 24px; }
-  h3 { font-size: 18px; }
-  h4 { font-size: 18px; }
-  h5 { font-size: 16px; }
-  h6 { font-size: 16px; }
+  h1 { font-size: 36px; } // note: github uses 2em, ~32px
+  h2 { font-size: 24px; } // note: github uses 1.5em, ~24px
+  h3 { font-size: 21px; } // note: github uses 1.25em, ~20px
+  h4 { font-size: 18px; } // note: github uses 1em, ~16px
+  h5 { font-size: 17px; } // note: github uses 0.875em, ~14px
+  h6 { font-size: 16px; } // note: github uses 0.85em, ~13.6px
```

Comment:
I suspect these are a bit close, but I'm not too worried.

From what I can tell wikipedia has the same size for h4 - h6

![image](https://github.com/user-attachments/assets/6c630d9a-7f68-4157-a11f-0e33af98a278)

with lines under h1 and h2, and h3 only being slightly larger.

Just saying it's an option to not have different sizes.

Reply:
I note that it might be more importnat to have a larger difference between h3 and h4, than having any difference between h4-h6

People rarely use h5 and h6.

If we wanted to make using h1-h6 usable, we should perhaps consider numbering headlines.

> # 1. h1
> ## 1.1 h2
> ### 1.1.1 h3
> #### 1.1.1.1 h4
> ##### 1.1.1.1.1 h5
> ###### 1.1.1.1.1.1 h6

But this can also be a bit degenerate in it's own right... Though we could put `1.1.1.1.1` in the margin and make it only show up when hovering a headline, or something like that.

Reply:
I think a table of contents hovering/updating/folding/unfolding as you scroll may be better than numbers.